### PR TITLE
Generate rancher-monitoring and rancher-logging version

### PR DESCRIPTION
**Pull request by w13915984028**
https://github.com/harvester/harvester/issues/6289

### CODE REVIEW

**Review by tserong — APPROVED** [review]
LGTM (just one little typo)

File: version_info [hunk 1]
```diff
@@ -5,4 +5,7 @@ PCIDEVICES_CONTROLLER_CHART_VERSION="0.4.0"
 PCIDEVICES_CONTROLLER_IMAGE="rancher/harvester-pcidevices:v0.4.0"
 HARVESTER_SEEDER_CHART_VERSION="0.3.0"
 HARVESTER_SEEDER_IMAGE="rancher/harvester-seeder:v0.3.0"
-NVIDIA_DRIVER_RUNTIME_CHART_VERSION="0.1.1"
\ No newline at end of file
+NVIDIA_DRIVER_RUNTIME_CHART_VERSION="0.1.1"
+RANCHER_LOGGING_CHART_VERSION="103.1.0+up4.4.0"
+RANCHER_MONITORING_CHART_VERSION="103.1.1+up45.31.1"
+HARVESTER_EVENTROUTER_FULL_TAG1="rancher/harvester-eventrouter:v0.3.1"
```
[the remark this answers is not in the document]
Comment (w13915984028) [reply version_info:11]: forgot to remove this test set :)

**Review by bk201 — APPROVED** [review]
LGTM, thanks!

**Comment by tserong** [comment]
@mergifyio backport v1.4

**Pull request merged**